### PR TITLE
Enable cache busting in production

### PR DESCRIPTION
**Pull request by 2e2a**
Enable the `ManifestStaticFilesStorage`\[1\] (non strict) on productions.

It adds hashes to all static files collected with collectstatic providing cache busting and allowing to cache them forever (needs to be configured on the web server).

The build server needs to be configured to run collectstatic twice. Once with the build configuration and once with the production configuration. Added the `collectstatic_prod` build step to our build script for this.

\[1\] https://docs.djangoproject.com/en/1.11/ref/contrib/staticfiles/#manifeststaticfilesstorage

### CODE REVIEW

**Comment by fuzzylogic2000** [comment]
To enable cachebusting, this PR and a change to the build settings is needed. We already changed the build settings, which will not enable the cache busting, but also not break anything before this is merged. We decided to merge this PR and deploy it after the idea challenge started and on a Monday or Tuesday. So the 4th of December would be a good date.

**Comment by slomo** [comment]
I would propose to not merge this and instead take the same approach as in OPIN: https://github.com/liqd/a4-opin/pull/1008

**Comment by slomo** [comment]
Okay I updated this to work with the https://github.com/liqd/adhocracy4/pull/219 but it requires a adhocracy update in ae.

**Comment by slomo** [comment]
Can be merged now into master.

**Review by fuzzylogic2000 — APPROVED** [review]

**Pull request merged**
